5-infrastructure/modules/base-env/outputs.tf

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  SQL Outputs
+*****************************************/
+
+output "sql_outputs" {
+  description = "Outputs for Cloud SQL instances"
+  value       = tomap({ for key, instance in module.sql : key => { "Instance Name" = instance.sql_instance_name, "Private IP Address" = instance.private_ip_address } })
+}
+
+/******************************************
+  Logging Outputs
+*****************************************/
+
+output "ops_log_export_map" {
+  description = "Outputs from the log export module"
```

Comment:
Are these used downstream?

Reply:
I can cut out logging entirely based on just downstream, though the module does create random identities, random sink name, random bucket name so left it in for user convenience

Reply:
yes lets only output values that are necessary downstream like (ACM/ASM install)